requirements/constraints.txt

```diff
@@ -9,3 +9,6 @@
 # linking to it here is good.
 
 Django<5
+
+# Common constraints for edx repos
```

Comment:
```suggestion
# Common constraints for Open edX repos
```
This is pretty handy but is there a reason we're adding it now?

Reply:
If we're going to start adopting this, I suggest adding it to the other EOX repositories if they don't already have it.

Reply:
I think this is the correct link: https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt

Reply:
Hi @mariajgrimaldi, thanks for the review!

> This is pretty handy but is there a reason we're adding it now?

I added it specifically because of a conflict in the versions of the dependencies. More info: https://github.com/eduNEXT/eox-tenant/actions/runs/9371722576/job/25801603124#step:6:145

> I think this is the correct URL: https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt, since all repos were transferred from edx to the openedx org.

Yes, you are right. I will update the link. Thanks!